src/Shared/Recruit.Vacancies.Client/Application/Commands/SubmitVacancyCommand.cs

```diff
@@ -1,13 +1,12 @@
-﻿using System;
-using Esfa.Recruit.Vacancies.Client.Domain.Entities;
+﻿using Esfa.Recruit.Vacancies.Client.Domain.Entities;
 using Esfa.Recruit.Vacancies.Client.Domain.Messaging;
 using MediatR;
 
 namespace Esfa.Recruit.Vacancies.Client.Application.Commands
 {
     public class SubmitVacancyCommand : ICommand, IRequest
     {
-        public Guid VacancyId { get;set; }
+        public Vacancy Vacancy { get;set; }
```

Comment:
It's a good point. Are you thinking of passing the id and the description into the  submit command? i.e. just the info required?

Reply:
Yep we can. It would mean we'd be dropping the validation on the employer description field on submit and relying on the Employer profile contents to be both populated and valid (which they should be). 

Everyone happy with that?

Reply:
Yep. That makes sense